src/examples/PictureJpg.cpp

```diff
@@ -60,7 +60,7 @@ void tvgDrawCmds(tvg::Canvas* canvas)
     file.close();
 
     auto picture = tvg::Picture::gen();
-    if (picture->load(data, size, "jpg", true) != tvg::Result::Success) {
+    if (picture->load(data, size, "jpg", "", true) != tvg::Result::Success) {
```

Comment:
How about declaring a macro for `""`?
```
#define TVG_NO_EXTERNAL_RESOURCE ""
```

Reply:
Well... I have no idea about it but I'd like to change string -> char*